pkg/agent/openflow/client.go

```diff
@@ -823,60 +793,113 @@ func (c *client) Initialize(roundInfo types.RoundInfo, nodeConfig *config.NodeCo
 	return connCh, c.initialize()
 }
 
-func (c *client) InstallExternalFlows(exceptCIDRs []net.IPNet) error {
-	localGatewayMAC := c.nodeConfig.GatewayConfig.MAC
+func (c *client) initializeFeatures() map[pipeline][]*featureTemplate {
+	var features []feature
```

Comment:
No need to define variable `features`, and you can use c.activeFeatures directly?

Reply:
You are right. I'll update this.